src/en/wip-storage.md

```diff
@@ -84,16 +86,27 @@ Charms may declare multiple types of storage, in which case they may all be spec
 juju deploy cs:~axwalk/postgresql --storage data=ebs,10G cache=ebs-ssd
 ```
 
-### Persistence (not working!)
+### Persistence (incomplete!)
+
+Some providers have the option to detach storage from the lifespan of the instance(s) which created/used it. This means that even after services have been removed, the storage and its contents still exist in your cloud, which may be useful for backup, recovery or transport purposes. Juju will not allow you to completely destroy an environment which contains such storage (except by using the --force option). To create persistent volumes, create a storage pool with the "persistent" attribute set to "true". Persistent volumes are currently supported by the EBS and Cinder storage providers. Currently, all Cinder volumes are considered persistent, whether the pool is configured as such.
+
+Juju does not currently provide any means of decomissioning persistent storage, so you must do this manually after force-destroying the environment. This will be rectified in a future version of Juju.
 
-Some providers have the option to detach storage from the lifespan of the instance(s) which created/used it. This means that even after services have been removed, the storage and its contents still exist in your cloud, which may be useful for backup, recovery or transport purposes. Juju will not allow you to completely destroy an environment which contains such storage (except by using the --force option). If this feature is enabled for storage pools, you must remember to decommission the storage manually.
+### Provider support
+
+All environment providers support the following storage providers:
+ - loop: block-type, creates a file in the agent data-dir and attaches a loop device to it. See the caveats section below for a comment on using the loop storage provider with local/LXC.
+ - rootfs: filesystem-type, creates a sub-directory in the agent's data-dir for the unit/charm to use
+ - tmpfs: filesystem-type, creates a tmpfs
 
-### Provider-specific options
+Additionally, native storage providers exist for the EC2 (ebs) and OpenStack (cinder).
 
-AWS/EC2
-OpenStack
-MAAS
+The EC2/EBS provider currently supports the following pool configuration attributes:
+ - volume-type: specifies the EBS volume type to create (gp2 (ssd), io1 (provisioned-iops), standard (magnetic)). By default, magnetic/standard volumes will be created. An 'ebs-ssd' pool is created in all EC2 environments, which defaults the volume-type to ssd/gp2.
+ - iops: the number of IOPS for provisioned-iops volume types.
```

Comment:
i think we could/should add a note that iops restrictions exist, see http://blah for more details